examples/twittermap/web/public/javascripts/common/services.js

```diff
@@ -473,17 +473,17 @@ angular.module('cloudberry.common', ['cloudberry.mapresultcache'])
             break;
           case "points":
             if(angular.isArray(result.value)) {
-              cloudberryService.tweetResult = result.value[0].slice(0, defaultSamplingSize - 1);
-              cloudberryService.pointsResult = result.value[0];
+              cloudberryService.commonTweetResult = result.value[0].slice(0, defaultSamplingSize - 1);
+              cloudberryService.pointmapMapResult = result.value[0];
```

Comment:
Whenever pointmap is used, let's change it to "pinmap".

Reply:
I will do all the renaming in a separate PR.